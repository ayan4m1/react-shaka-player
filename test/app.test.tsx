import * as React from "react";
import * as renderer from "react-test-renderer";
import * as Shaka from "shaka-player/dist/shaka-player.ui";
import Player from "../src";

it("Player renders correctly", () => {
  const props = { className: "video-test" };
  const tree = renderer.create(<Player {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});

jest.mock("shaka-player/dist/shaka-player.ui");

it("Shaka player called correctly", () => {
  const component = renderer.create(<Player />);
  expect(Shaka.Player).toHaveBeenCalled();
  expect(Shaka.ui.Overlay).toHaveBeenCalled();
});
