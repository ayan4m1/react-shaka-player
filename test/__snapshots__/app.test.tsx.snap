// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player renders correctly 1`] = `
<div
  className="video-test"
>
  <video
    className="video-test"
    style={
      Object {
        "maxWidth": "100%",
        "width": "100%",
      }
    }
  />
</div>
`;
